src/test/java/org/apache/commons/imaging/formats/tiff/TiffShortIntRoundTripTest.java

```diff
@@ -0,0 +1,213 @@

+        try (FileOutputStream fos = new FileOutputStream(outputFile);
+            BufferedOutputStream bos = new BufferedOutputStream(fos)) {
+            final TiffImageWriterLossy writer = new TiffImageWriterLossy(byteOrder);
+            writer.write(bos, outputSet);
+            bos.flush();
+        }
+        return outputFile;
+    }

+    /**
+     * Gets the bytes for output for a 16 bit floating point format. Note that
+     * this method operates over "blocks" of data which may represent either
+     * TIFF Strips or Tiles. When processing strips, there is always one column
+     * of blocks and each strip is exactly the full width of the image. When
+     * processing tiles, there may be one or more columns of blocks and the
+     * block coverage may extend beyond both the last row and last column.
+     *
+     * @param s an array of the grid of output values in row major order
+     * @param width the width of the overall image
+     * @param height the height of the overall image
+     * @param nRowsInBlock the number of rows in the Strip or Tile
+     * @param nColsInBlock the number of columns in the Strip or Tile
+     * @param byteOrder little endian or big endian
+     * @return a two-dimensional array of bytes dimensioned by the number of blocks and samples
+     */
+    private byte[][] getBytesForOutput16(
+        final short[] s,
+        final int width, final int height,
+        final int nRowsInBlock, final int nColsInBlock,
+        final ByteOrder byteOrder) {
+        final int nColsOfBlocks = (width + nColsInBlock - 1) / nColsInBlock;
+        final int nRowsOfBlocks = (height + nRowsInBlock + 1) / nRowsInBlock;
+        final int bytesPerPixel = 2;
+        final int nBlocks = nRowsOfBlocks * nColsOfBlocks;
+        final int nBytesInBlock = bytesPerPixel * nRowsInBlock * nColsInBlock;
+        final byte[][] blocks = new byte[nBlocks][nBytesInBlock];
+        for (int i = 0; i < height; i++) {
+            final int blockRow = i / nRowsInBlock;
+            final int rowInBlock = i - blockRow * nRowsInBlock;
+            final int blockOffset = rowInBlock * nColsInBlock;
+            for (int j = 0; j < width; j++) {
+                final int value = s[i * width + j];
+                final int blockCol = j / nColsInBlock;
+                final int colInBlock = j - blockCol * nColsInBlock;
+                final int index = blockOffset + colInBlock;
+                final int offset = index * bytesPerPixel;
+                final byte[] b = blocks[blockRow * nColsOfBlocks + blockCol];
+                if (byteOrder == ByteOrder.LITTLE_ENDIAN) {
+                    b[offset] = (byte) (value & 0xff);
+                    b[offset + 1] = (byte) ((value >> 8) & 0xff);
+                } else {
+                    b[offset] = (byte) ((value >> 8) & 0xff);
+                    b[offset + 1] = (byte) (value & 0xff);
+                }
+            }
+        }

+        return blocks;
+    }


```

Comment:
I think this symbol is a Windows CRLF, but I think the project is using LF (linux). It occurs in other parts of this file :point_up: 

If you'd like to configure your git, there's a way to tell it to default to Linux, to Windows, to what the project is using, etc... https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

Reply:
Yes, that's definitely how Linux will show a carriage return (ASCII code 13, and M is the 13 letter in the English alphabet).

Is the Linux form (just the Linefeed) is the standard file encoding in Commons Imaging?  If so, I will reconfigure my git utility that way.

Reply:
done